mlora/utils/deletefile.py

```diff
@@ -0,0 +1,25 @@
+import os
+import shutil
+import os.path
+
+def delete_files_in_folder(folder_path):
+    try:
+        # Iterate through each file in the directory
+        for item in os.listdir(folder_path):
+            item_path = os.path.join(folder_path, item)
+            if os.path.isfile(item_path):  # If it's a file, delete it
```

Comment:
i think we do not need those comment, just known what you want to do by reading those code.

Reply:
ok